tests/unit/channel_test.py

```diff
@@ -33,19 +33,21 @@ def ssh_channel(ssh_client_session):
         chan.close()
 
 
-@pytest.mark.xfail(
-    reason='This test causes SEGFAULT, flakily. '
-    'Ref: https://github.com/ansible/pylibssh/issues/57',
-    strict=False,
-)
+def exec_second_command(ssh_channel):
+    """Call exec_command() from different context in the call stack."""
+    u_cmd_out = ssh_channel.exec_command('echo -n Hello Again').stdout.decode()
+    assert u_cmd_out == u'Hello Again'  # noqa: WPS302
+
+
 @pytest.mark.forked
 def test_exec_command(ssh_channel):
     """Test getting the output of a remotely executed command."""
     u_cmd_out = ssh_channel.exec_command('echo -n Hello World').stdout.decode()
     assert u_cmd_out == u'Hello World'  # noqa: WPS302
     # Test that repeated calls to exec_command do not segfault.
-    u_cmd_out = ssh_channel.exec_command('echo -n Hello Again').stdout.decode()
-    assert u_cmd_out == u'Hello Again'  # noqa: WPS302
+
+    # randomize the stack a bit more by calling this from yet another function
```

Comment:
Is this accurate enough?
```suggestion
    # NOTE: Call `exec_command()` once again from another function to
    # NOTE: force it to happen in another place of the call stack,
    # NOTE: making sure that the context is different from one in this
    # NOTE: this test function. The resulting call stack will end up
    # NOTE: being more random.
```

Reply:
Looks good. Applied.